onadata/libs/utils/logger_tools.py

```diff
@@ -62,7 +62,9 @@
 DEFAULT_CONTENT_LENGTH = settings.DEFAULT_CONTENT_LENGTH
 REQUIRED_ENCRYPTED_FILE_ELEMENTS = [
     "{http://www.opendatakit.org/xforms/encrypted}base64EncryptedKey",
-    "{http://www.opendatakit.org/xforms/encrypted}encryptedXmlFile"]
+    "{http://www.opendatakit.org/xforms/encrypted}encryptedXmlFile",
+    "{http://opendatakit.org/submissions}base64EncryptedKey",
+    "{http://opendatakit.org/submissions}encryptedXmlFile"]
```

Comment:
Are the new tags unique to only Enketo ? 

_There is a [check](https://github.com/onaio/onadata/blob/fd58c7e26b5c136b8b0915c3714f1beaebdb7bcb/onadata/libs/utils/logger_tools.py#L320-L325) to confirm that only `2` of the tags are present hopefully there isn't a scenario where more than two of these tags can be present in an XML Submission_

Reply:
The two are the only elements we need for this encryption feature

Reply:
ODK Collect shares the same elements ?

Reply:
They use the same elements but different namespaces. For odk it is `{http://www.opendatakit.org/xforms/encrypted}` while for enketo is `{http://opendatakit.org/submissions}`

Reply:
Ohh cool, makes sense